File: HERENCIA-PYTHON/Clases/Vehiculo.py
class Vehiculo:
    def __init__(self, Fecha_Fabricacion, VIN_Chasis, VIN_Motor):
        self.Fecha = Fecha_Fabricacion
        self.VIN_Chasis = VIN_Chasis
        self.VIN_Motor = VIN_Motor

    def get_Fecha_Fabricacion(self):
        return self.Fecha   
    
    def get_VIN_Chasis(self):
        return self.VIN_Chasis
    
    def get_VIN_Motor(self):
        return self.VIN_Motor
    
    

File: src/Clases/Vehiculo.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package Clases;

/**
 *
 * @author Didier
 */
public class Vehiculo {
    private String fechaFabricacion;
    private String vinChasis;
    private String vinMotor;

    
    public Vehiculo(String fechaFabricacion, String vinChasis, String vinMotor) {
        this.fechaFabricacion = fechaFabricacion;
        this.vinChasis = vinChasis;
        this.vinMotor = vinMotor;
    }

    
    public String getFechaFabricacion() {
        return fechaFabricacion;
    }

    public String getVinChasis() {
        return vinChasis;
    }

    public String getVinMotor() {
        return vinMotor;
    }
    
}
